app/config/errors.php

```diff
@@ -643,7 +643,7 @@
     ],
     Exception::ROUTER_DOMAIN_NOT_CONFIGURED => [
         'name' => Exception::ROUTER_DOMAIN_NOT_CONFIGURED,
-        'description' => 'Domain environment variables not configured. Please configure domain environment variables before using Appwrite outside of localhost.',
+        'description' => '_APP_DOMAIN, _APP_DOMAIN_TARGET, and _APP_DOMAIN_FUNCTIONS environment variables have not been configured. Please configure the domain environment variables before accessing the Appwrite Console via anything other than localhost.',
```

Comment:
```suggestion
        'description' => '_APP_DOMAIN, _APP_DOMAIN_TARGET, and _APP_DOMAIN_FUNCTIONS environment variables have not been configured. Please configure the domain environment variables before accessing the Appwrite Console via any hostname other than localhost.',
```

Reply:
If they're accessing Appwrite via IP, they'll need to set the _APP_DOMAIN env var to that IP too.

Reply:
Can we make that more explicit? Because the env variable `DOMAIN` does not reflect that it could also be an IP.

I would also make sure we have this somewhere in the docs ? @gewenyu99

Reply:
K I'll update the text to

> _APP_DOMAIN, _APP_DOMAIN_TARGET, and _APP_DOMAIN_FUNCTIONS environment variables have not been configured. Please configure the domain environment variables before accessing the Appwrite Console via any hostname or IP address other than localhost.

Reply:
Per Vincent's suggestion, I'll change it to

> _APP_DOMAIN, _APP_DOMAIN_TARGET, and _APP_DOMAIN_FUNCTIONS environment variables have not been configured. Please configure the domain environment variables before accessing the Appwrite Console via any IP address or hostname other than localhost. This value could be an IP like 203.0.113.0 or a hostname like example.com.

Reply:
@christyjacob4 @gewenyu99, updated.